invoker/rh-changes/pom.xml

```diff
@@ -41,6 +41,18 @@
                 <type>tar.gz</type>
             </dependency>
             <dependency>
+                <groupId>org.eclipse.che.plugin</groupId>
```

Comment:
Here the `groupId` should be  `com.redhat.che` IMO (see the review comment on https://github.com/redhat-developer/rh-che/pull/39/files#diff-a0951a27f025f1f7de5ead9e60153ab7).

As for the version, there's no reason why we should use the upstream Che version. No reason, as well, that it would follow the version of the RH distribution that derives from the upstream Che version. Beyesian plugins, like Keycloak plugins, are RH-only standalone components (added into the RH Che distribution) that can have their own version.
So it seems to me that we should add a property:
```
        <bayesian.plugins.version>1.0.0-SNAPSHOT</bayesian.plugins.version>
```
as done for the keycloak plugins, and use it here in the dependency management.